src/runtime_src/core/common/utils.cpp

```diff
@@ -217,4 +219,110 @@ issue_id()
   return id++;
 }
 
+int
+parse_p2p_config(const std::vector<std::string> config, std::string &msg)
```

Comment:
```suggestion
parse_p2p_config(const std::vector<std::string>& config, std::string& msg)
```
Perhaps you could also return a `pair` of `int` and `std::string` instead?

Reply:
Add some comments on the declaration of the function too?